src/components/User/Users/components/UserDetails/UserDetails.Request.vue

```diff
@@ -91,42 +197,88 @@ export default {
   components: {
     Modal,
     TextField,
+    TickField,
   },
 
   props: {
     requestToReview: {
       type: ChangeRoleRequest,
-      default: _ => new ChangeRoleRequest({}),
+      default: () => new ChangeRoleRequest({}),
     },
     latestApprovedRequest: {
       type: ChangeRoleRequest,
-      default: _ => new ChangeRoleRequest({}),
+      default: () => new ChangeRoleRequest({}),
     },
     user: {
       type: Object,
-      default: _ => ({}),
+      default: () => ({}),
     },
   },
 
   data () {
     return {
+      tasksToAdd: 0,
+      tasksToRemove: 0,
+
       rejectForm: {
         reason: '',
         isShown: false,
       },
       isShownAdvanced: false,
       isPending: false,
+
+      taskValues: {
+        submitAutoVerification: null,
+        completeAutoVerification: null,
+        manualReviewRequired: null,
+        default: 1,
+      },
     }
   },
 
+  async created () {
+    await this.loadTaskValues()
+    this.tasksToRemove = this.requestToReview.pendingTasks
+  },
+
   methods: {
+    async getTaskFromKv (key) {
+      const { data } = await ApiCallerFactory
+        .createCallerInstance()
+        .get(`v3/key_values/${key}`)
+
+      return data.value.u32
+    },
```

Comment:
can we collect the tasks at the start of the application? just like we do for other key-values